koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -243,8 +243,8 @@ def get_ocp_infrastructure_map_trino(self, start_date, end_date, **kwargs):  # n
                 for entry in results:
                     # This dictionary is keyed on an OpenShift provider UUID
                     # and the tuple contains
-                    # (Infrastructure Provider UUID, Infrastructure Provider Type)
-                    db_results[entry[0]] = (entry[1], entry[2])
+                    # (Infra Provider UUID, Infra Provider Type, Infra account number, Infra region)
```

Comment:
Up on line 161, I think we still use that following this code path:

ocp_on_cloud_infra_map = updater.get_openshift_on_cloud_infra_map(start_date, end_date, tracing_id)